### PR TITLE
[WIP] Adding optuna example

### PR DESCRIPTION
**Pull request by jorshi**
An example of using optuna with multi-scale STFT loss to try to estimate parameters to match a target sound.

### CODE REVIEW

File: torchsynth/loss.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import torch
+
+from torch import nn
+from torch.nn import functional as F
+
+"""
+Multi-scale STFT loss in torch from https://github.com/vwcgaming/TTS
+"""
```
Comment (jorshi) [inline torchsynth/loss.py:8]: Just copied this code from that repo

**Comment by jorshi** [comment]
We've got optuna stuff working for experiments. Closing this for now.
